spec/rubocop/cop/rspec/file_path_spec.rb

```diff
@@ -204,6 +204,14 @@
     RUBY
   end
 
+  # RSpec/FilePath runs on all files - not only **/*_spec.rb
```

Comment:
Wondering, why so?
For some reason `relevant_rubocop_rspec_file?` evaluates to `true` for it.

Reply:
Ah, I see it
```ruby
        def relevant_rubocop_rspec_file?(_file)
          true
        end
```